utils/date.ts

```diff
@@ -0,0 +1,2 @@
+export const formatDate = (date: string | number | Date) =>
+  new Intl.DateTimeFormat('en-GB').format(new Date(date));
```

Comment:
Let use the browser's locale here instead of a hard coded UK region

Reply:
Also, make it configurable, should be another param of the `formatDate` function just in case.